pkg/cmd/render/render.go

```diff
@@ -63,6 +84,41 @@ func (r *renderOpts) Validate() error {
 		return err
 	}
 
+	clusterInfrastructureFileLen := len(r.clusterInfrastructureFile)
+	cloudProviderConfigInputFileLen := len(r.cloudProviderConfigInputFile)
+	cloudProviderConfigOutputFileLine := len(r.cloudProviderConfigOutputFile)
```

Comment:
I don't think these make it any easier to read. Just inline the use of len and drop the vars.

Reply:
Removed.